libraries/AdaptiveExpressions/ExpressionFunctions.cs

```diff
@@ -772,7 +772,7 @@ public static ExpressionEvaluator TimeTransform(string type, Func<DateTime, int,
                         }
                         else
                         {
-                            error = $"{expr} could not be evaluated";
+                            error = $"{expr} should contain an ISO format timestamp and an integer of time interval.";
```

Comment:
>of [](start = 99, length = 2)

I would drop this word.

Reply:
changed to $"{expr} should contain an ISO format timestamp and and a time interval integer."